Bindings/Packages/com.virtualmaker.bindings/Runtime/Bindings.InputActions.cs

```diff
@@ -130,10 +142,18 @@ public void On<T>(InputActionReference inputActionReference, Action<T> onPerform
             void OnActionCanceled(InputAction.CallbackContext ctx) => onCancelled(ctx.ReadValue<T>());
         }
 
+        [Obsolete("Use 'OnInputAction' instead.", false)]
```

Comment:
Would rather not have obsolete versions until we have an external team using this pkg